src/main/java/seedu/address/storage/JsonAdaptedInterview.java

```diff
@@ -0,0 +1,165 @@
+package seedu.address.storage;
```

Comment:
From the looks of this file, I think it's still a WIP?

Lots of complaints from codecov 💢 let me take a look at how `JsonAdaptedPerson` is tested.

Think we need to discuss about the JSON format for the `Interview`.

I believe we need only `studentId` and an array of the `schedule`. And we probably only need to test these 2 fields for this file.

To KIV.

Again, good job leon!

Reply:
Yes indeed Dom - great catch! While working on this I was wondering if it is possible to just store the `studentId` but ensure it works in the `toModelType()` method as well -> meaning to create an `Candidate` object from just the `studentId` field. However my brain was at its limit already so I left it as it is 😅